pythonbits/bb.py

```diff
@@ -263,7 +263,13 @@ def _render_tags(self):
         tags += [a['name']
                  for a in self['summary']['cast'][:n]
                  if a['name']]
-        return ",".join(format_tag(tag) for tag in tags)
+
+        # Maximum tags length is 200 characters
+        def tags_string(tags):
+            return ",".join(format_tag(tag) for tag in tags)
+        while len(tags_string(tags)) > 200:
+            del tags[-1]
+        return tags_string(tags)
```

Comment:
not quite optimal code but it's very expressive

something like

    ftd = ",".join(format_tag(tag) for tag in tags)
    if len(ftd) > 200:
        i = ftd[:201][::-1].find(',')
        ftd = ftd[:200-i]
    return ftd

would offer a 2x speedup both for short and long tag lists